README.md

```diff
@@ -306,7 +306,7 @@ function user_login(self)
     local user = box.select(users, 1, login, password)
     if user ~= nil then
         return self:redirect_to('/'):
-            set_cookie({ name = 'uid', value = user[0], expires = '+1y' })
+            setcookie({ name = 'uid', value = user[0], expires = '+1y' })
```

Comment:
1. The description above lie about we should use `self:cookie(cookies)` (since 2fb0d0e91090efdc653063cd68ec57053c5bba14).
2. This should be before `return`.
3. Colon at the line above.
4. It seems it is the method of a response object and not a request object.

So this seems to be broken in more ways then fixed by this PR and the entire section need to be verified.